packages/Search/stories/examples/helpers.js

```diff
@@ -0,0 +1,9 @@
+export async function fetchAPI(term, offset = null, limit = 20) {
+  const url = "https://gateway.marvel.com:443/v1/public/characters?nameStartsWith=";
```

Comment:
Didn't we find an alternative before so we didn't have to hit an external API?  (If I recall, some tests were failing because of it... not that this has any tests)

Reply:
Do you have an alternative API that we could use? this is quite simple and mostly I'm using it to demonstrated a case. 

I wonder if would be beneficial to create our own testing API for our component :P

Reply:
Oh here is the `Issue` i made before to replace an external API: https://github.com/acl-services/paprika/issues/782.
I'll just add the `Search` component to that `Issue`, so when it is done, it is fixed here too.